packages/react-core/src/modules/slack/hooks/useSlackAuth.ts

```diff
@@ -9,8 +9,7 @@ const DEFAULT_SLACK_SCOPES = [
   "chat:write",
   "chat:write.public",
   "channels:read",
-  "groups:read",
-  "groups:write",
```

Comment:
I found out while writing docs this write scope is unnecessary, it allows the bot to create new private channels which we're not using